contextily/place.py

```diff
@@ -9,9 +9,8 @@
 from ._providers import TileProvider
 
 # Set user ID for Nominatim
-rng = np.random.default_rng()
-val = rng.integers(1000000)
-gp.geocoders.options.default_user_agent = f"contextily_user_{val}"
+_val = np.random.randint(1000000)
```

Comment:
This sounds good and more backwards compatible. I tried to go with the numpy recommendation but if it's not a widespread approach yet and fails on earlier versions, let's go with `randint`